Cargo.toml

```diff
@@ -181,6 +181,7 @@ unstable-ext = ["clap_builder/unstable-ext"]
 unstable-styles = ["clap_builder/unstable-styles"]  # deprecated
 unstable-derive-ui-tests = []
 unstable-markdown = ["clap_derive/unstable-markdown"]
+unstable-command-headings =["unstable-command-headings"]
```

Comment:
Why are we starting this off as unstable?

Reply:
I thought that was the convention.  I can rename the the feature flag or remove it if it is unnecessary.